apiary-metastore-listener/README.md

```diff
@@ -25,20 +26,29 @@ topic:
 |`eventType`|String|Always|One of: CREATE_TABLE, DROP_TABLE, ALTER_TABLE, ADD_PARTITION, DROP_PARTITION, ALTER_PARTITION| 
 |`dbName`|String|Always|The Hive database name|
 |`tableName`|String|Always|The Hive table name|
+|`tableLocation`|String|In all events except INSERT|The Hive table location|
+|`tableParameters`|String|In all events except INSERT|The Hive table parameters|
 |`oldTableName`|String|Only when `eventType` is ALTER_TABLE|The old Hive table name|
 |`partition`|String|Only when the `eventType` is one of ADD_PARTITION, DROP_PARTITION, ALTER_PARTITION|The Hive partition values|
+|`partitionLocation`|String|Only when the `eventType` is one of ADD_PARTITION, DROP_PARTITION, ALTER_PARTITION|The Hive partition location|
 |`oldPartition`|String|Only when the `eventType` is ALTER_PARTITION|The old Hive partition values|
+|`oldPartitionLocation`|String|Only when the `eventType` is ALTER_PARTITION|The old Hive partition location|
 
 ### Example Messages
 
 #### Create table
-The following shows an example JSON message representing a "CREATE_TABLE" event:
+The following shows an example JSON message representing a "CREATE_TABLE" event when environment variable TABLE_PARAM_FILTER is set to `my_var.*`:
 
 	{
       "protocolVersion": "1.0",
       "eventType": "CREATE_TABLE",
       "dbName": "some_db",
-      "tableName": "some_table"
+      "tableName": "some_table",
+      "tableLocation": "s3://table_location",
+      "tableParameters": {
+          "my_var_one": "val_one",
+          "my_var_two": "val_two"
+       }
```

Comment:
Probably not your fault but the indents of this example message are different to the others so this looks a bit weird, can you reset this whole block to match the ones below please?

Reply:
fixed.